QuizAppNew/Model/QuestionModel.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  QuestionModel.swift
+//  QuizAppNew
+//
+//  Created by Jannatun Nahar Papia on 23/8/22.
+//
+
+import Foundation
+
+//
+//    RootClass.swift
+//    Model file generated using JSONExport: https://github.com/Ahmed-Ali/JSONExport
+
+import Foundation
+
+struct QuestionModel : Codable {
+    
+    let answer : String?
+    let category : String?
+    let image : String?
+    let option1 : String?
+    let option2 : String?
+    let option3 : String?
+    let option4 : String?
+    let qustion : String?
+    let sl : Int?
+    
+    
+    enum CodingKeys: String, CodingKey {
+        case answer = "answer"
+        case category = "category"
+        case image = "image"
+        case option1 = "option1"
+        case option2 = "option2"
+        case option3 = "option3"
+        case option4 = "option4"
+        case qustion = "qustion"
+        case sl = "sl"
+    }
+    init(from decoder: Decoder) throws {
+        let values = try decoder.container(keyedBy: CodingKeys.self)
+        answer = try values.decodeIfPresent(String.self, forKey: .answer)
+        category = try values.decodeIfPresent(String.self, forKey: .category)
+        image = try values.decodeIfPresent(String.self, forKey: .image)
+        option1 = try values.decodeIfPresent(String.self, forKey: .option1)
+        option2 = try values.decodeIfPresent(String.self, forKey: .option2)
+        option3 = try values.decodeIfPresent(String.self, forKey: .option3)
+        option4 = try values.decodeIfPresent(String.self, forKey: .option4)
+        qustion = try values.decodeIfPresent(String.self, forKey: .qustion)
+        sl = try values.decodeIfPresent(Int.self, forKey: .sl)
+    }
+}
+
+
+var questions = [QuestionModel]()
+
+func localCalling(link: String) {
+
+    if let path = Bundle.main.path(forResource: "data", ofType: "json") {
+        do {
+              let data = try Data(contentsOf: URL(fileURLWithPath: path), options: .mappedIfSafe)
+            let _posts = try? JSONDecoder().decode([QuestionModel].self, from: data)
+                
+            for item in _posts! {
+                print(item.qustion)
+            }
+
+        } catch {
+               // handle error
+              //      print("error")
+          }
+    }
+}
+
```

Comment:
I am not sure why you implementing JSON load here. I told you several times inside the **onAppear** and even commented there.

Reply:
this comment helps me lot